discount/src/main/java/com/kyriba/conference/discount/service/DiscountService.java

```diff
@@ -0,0 +1,22 @@
+package com.kyriba.conference.discount.service;
+
+import com.kyriba.conference.discount.api.dto.DiscountExternalDto;
```

Comment:
Service could not use DTO from API level. Move them to domain model.

Reply:
I just convert Dto's to Entities at service level ... where do you think I should do this?

Reply:
service package doesn't know about API package and could not depend on it. At the same time, api package could reuse business logic DTOs to speed up.